API.md

```diff
@@ -1,5 +1,6 @@
 # Table of Contents
 
+- [Table of Contents](#table-of-contents)
```

Comment:
It's a good addition. Leave it as it is now, but it's better not to mix purposes of PR.

Reply:
Oh, vscode did this automatically, didn't mean to do that.